src/components/layout/Info.js

```diff
@@ -15,9 +15,9 @@ const Info = ({ onToggleModalVisibility }) => {
   
   return (
     <div className="info">
-      {/* <OverlayTrigger placement="bottom" overlay={infoTooltip} delayHide={150} delayShow={300}>
-        <FaInfoCircle onClick={onClose} size={25}/>
-      </OverlayTrigger> */}
+      {/* <OverlayTrigger placement="bottom" overlay={infoTooltip} delayHide={150} delayShow={300}> */}
+      {/* </OverlayTrigger> */}
```

Comment:
@jimmcgaw Leaving this in as a reminder to replace this?

Reply:
Aye. Going to see about incrementally upgrading react-bootstrap and see if we can get the same component without compatibility issues with react. If that won't work, then I'll devise some other tool to accomplish the same thing.

Reply:
Cool, sounds good. I don't know if this is any good, but... it does exist :sweat_smile: https://github.com/ReactTooltip/react-tooltip/

Reply:
Additional options: https://reactscript.com/best-tooltip-component/

But using `react-bootstrap` sounds good to me as long as it works well with the latest version of React.